graphql.config.json

```diff
@@ -1,10 +1,9 @@
 {
-  "schema": "https://www.hylo.com/noo/graphql",
-  "documents": "src/components/**/*.js",
-  "extension": {
+  "schema": "http://localhost:3001/noo/graphql",
+  "extensions": {
     "endpoints": {
       "default": {
-        "url": "https://www.hylo.com/noo/graphql"
+        "url": "http://localhost:3001/noo/graphql"
```

Comment:
what is this for? its not needed on production?

Reply:
"GraphQL configs related to VS Code GraphQL extensions" no, nothing production related